grants_tagger/tag_grants_with_mesh.py

```diff
@@ -0,0 +1,67 @@
+"""
+Tag grants with MeSH tags
+"""
+from argparse import ArgumentParser
+from pathlib import Path
+import csv
+
+from grants_tagger.predict_mesh import predict_mesh_tags as predict_tags
+
+MODEL_PATH = "models/disease_mesh_tfidf-svm-2020.07.0/"
```

Comment:
perhaps receive those as a default value argument in ArgParser?

Reply:
possibly better yes

Reply:
was just keeping in line with science tagger that hard codes the model that we use in production which is packed in the wheel. in this case we do not pack the model, it makes more sense to have it as an arg

Reply:
a7fa33452ac237d6f2c7113d410967fd04046582